src/stacker-responsibilities.md

```diff
@@ -1,8 +1,21 @@
 # Stacker responsibilities
-The stackers are gaining a lot of responsibilities in sBTC. This chapter should outline the new role of the stackers, and also provide a high level interaction to further details on how this is implemented.
+One of the most significant changes to accomodate the sBTC design is that Stackers must now perform active work to continue receiving PoX rewards. Stackers collectively maintain a BTC wallet for the duration of each reward cycle in which their STX are locked. This chapter outlines the new role of the stackers, and how they interact with each other to fulfill their duties.
 
-Important areas to introduce:
-- The sBTC wallet and how stackers maintain it (introducing FROST)
-- How stackers communicate (Signing protocol + StackerDB)
+- sBTC Wallet address generation
+- sBTC Wallet handoff
+- sBTC Withdrawal fulfillment
 
-TODO: [#10](https://github.com/stacks-network/sbtc-docs/issues/10)
+## sBTC wallet address generation
+Stackers in sBTC operate in Reward cycles similar to previous versions of [PoX](https://github.com/stacksgov/sips/blob/main/sips/sip-007/sip-007-stacking-consensus.md). However, the reward cycle has been modified to consist of three phases: prepare (80 blocks), handoff (20 blocks), and reward (2000 blocks).
```

Comment:
One open question going forward is how these phases will look after the nakamoto consensus rules takes effect https://github.com/stacks-network/stacks/blob/master/sBTC/nakamoto.md